modules/rostests/kmtests/ntos_ob/ObInsert.c

```diff
@@ -0,0 +1,651 @@
+/*
+ * PROJECT:         ReactOS kernel-mode tests
+ * LICENSE:         GPLv2+ - See COPYING in the top level directory
+ * PURPOSE:         Kernel-Mode Test Suite - ObInsertObject test
+ * PROGRAMMER:      Gleb Surikov <glebs.surikovs@gmail.com>
+ */
+
+#include <kmt_test.h>
+
+#define NDEBUG
+#include <debug.h>
+
+typedef struct DUMMY_TYPE
+{
+    ULONG Data;
+} DUMMY_TYPE, *PDUMMY_TYPE;
+
+static HANDLE DirectoryHandle;
+static UNICODE_STRING DirectoryName;
+
+static POBJECT_TYPE DummyType;
+static UNICODE_STRING DummyTypeName;
+static OBJECT_TYPE_INITIALIZER ObjectTypeInitializer;
+static OBJECT_ATTRIBUTES DummyObjectAttributes;
+
+static
+VOID
+NTAPI
+DumpProc(
+    IN PVOID Object,
+    IN POB_DUMP_CONTROL DumpControl
+)
```

Comment:
Closing parenthesis on the same line, preferably:
```suggestion
    IN POB_DUMP_CONTROL DumpControl)
```